elements/pfe-button/demo/pfe-button.html

```diff
@@ -47,8 +47,8 @@ <h2>Disabled</h2>
   <pfe-button disabled warning><button>Warning disabled</button></pfe-button>
 
   <h3>Link variant</h3>
-  <pfe-button disabled variant="link" icon="fa-plus-circle"><button>Link disabled</button></pfe-button>
-  <pfe-button disabled variant="link" icon="fa-external-link-square" icon-position="right"><button>Link disabled</button></pfe-button>
+  <pfe-button disabled variant="link" icon="plus-circle"><button>Link disabled</button></pfe-button>
+  <pfe-button disabled variant="link" icon="link" icon-position="right"><button>Link disabled</button></pfe-button>
```

Comment:
Do we think it would be worth giving them the option to also determine how the icon loads since we now have the loading option?  For example, with button you can now do something like 

```html
<pfe-button variant="link" icon="attention-bell" icon-set="patternfly" icon-position="right">
    <button>Bell</button>
  </pfe-button>
```

But we could also support something like 

```html
<pfe-button variant="link" icon="attention-bell" icon-set="patternfly" icon-position="right" icon-loading="eager">
    <button>Bell</button>
  </pfe-button>
```

Reply:
Perhaps it can wait until a user asks for it